custom_components/checkwatt/__init__.py

```diff
@@ -256,6 +270,46 @@ async def _async_update_data(self) -> CheckwattResp:
                     resp["spot_price"] = cw_inst.get_spot_price_excl_vat(time_hour)
                     resp["price_zone"] = cw_inst.price_zone
 
+                # Check if FCR-D State has changed and dispatch it ACTIVATED/ DEACTIVATED
+                old_state = self.fcrd_state
+                new_state = cw_inst.fcrd_state
+
+                # During test, toggle (every minute)
+                if BASIC_TEST is True:
+                    if old_state == "ACTIVATED":
+                        new_state = "DEACTIVATE"
```

Comment:
Saknas ett D?

Reply:
I think its correct, but I need to check my logbook again. It comes from EIB.